colors.go

```diff
@@ -313,27 +323,31 @@ func (sm styleMap) get(f *file) tcell.Style {
 		key = "ex"
 	}
 
-	if val, ok := sm[key]; ok {
+	if f.linkState == working && !sm.useLinkTarget {
+		key = "ln"
+	}
```

Comment:
IIRC `switch` statements can use conditional expressions and will simply use the first matching one, so you should just be able to include this in the `switch` statement above.

Reply:
I didn't know that about Go switch statements, thanks!